src/SecTester.Scan/DefaultScans.cs

```diff
@@ -0,0 +1,111 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using SecTester.Bus.Dispatchers;
+using SecTester.Bus.Extensions;
+using SecTester.Core;
+using SecTester.Core.Bus;
+using SecTester.Core.Exceptions;
+using SecTester.Scan.CI;
+using SecTester.Scan.Commands;
+using SecTester.Scan.Models;
+
+namespace SecTester.Scan;
+
+internal class DefaultScans : Scans
```

Comment:
It is not clear to me, why commands are public, but this one is internal. It should be the opposite.